File: src/main/java/com/avisys/cim/services/CustomerService.java
package com.avisys.cim.services;

import java.util.List;
import java.util.Optional;

import com.avisys.cim.entities.Customer;

public interface CustomerService {
	
	
	public List<Customer> getCustomers(String firstName, String lastName, String mobileNumber);
	
	public Customer addCustomer(Customer customer);
	
	public String deleteCustomerByMobileNumber(Customer customer, String mobileNumber);
	
	
	public Customer getCustomer(Long customerId);

}

File: src/services/customerService.js
import axiosServ from "./axiosServ"


const getCustomer = ()=> {
    
    return axiosServ.get("/customer");

}

const filCustomer = (firstName, lastName ,mobileNumber)=> {
   

    // if(firstName == "") {firstName = am}
    // if(lastName == "") {lastName = am}
    // if(mobileNumber == "") {mobileNumber = am}
    return axiosServ.get("/customer",{
        params: {
          firstName: firstName,
          lastName: lastName,
          mobileNumber: mobileNumber,
        },
      });

}
export default {getCustomer, filCustomer}